src/common/hooks/useOnboard.ts

```diff
@@ -43,85 +24,6 @@ export default function useOnboard() {
     disconnect,
   } = context;
 
-  // When network changes, reconnect
-  useEffect(() => {
-    if (initOnboard) {
-      // These are optional callbacks to be passed into onboard.
-      const subscriptions = {
-        address: (addr: string | null) => {
-          dispatch({ type: actions.SET_ADDRESS, payload: addr });
-        },
-        network: async (networkId: any) => {
-          if (networkId) {
-            dispatch({
-              type: actions.SET_NETWORK,
-              payload: {
-                chainId: networkId,
-                name: getNetworkName(networkId as ChainId),
-              },
-            });
-          } else {
-            dispatch({
-              type: actions.SET_NETWORK,
-              payload: null,
-            });
-          }
-
-          // Notify.js doesn't work on ganache anyway (see: docs).
-          // So don't bother initializing it on test.
-          if (process.env.REACT_APP_CURRENT_ENV !== "test") {
-            if (networkId) {
-              dispatch({
-                type: actions.SET_NOTIFY,
-                payload: Notify({
-                  dappId: process.env.REACT_APP_PUBLIC_ONBOARD_API_KEY, // [String] The API key created by step one above
-                  networkId, // [Integer] The Ethereum network ID your Dapp uses.
-                  desktopPosition: "topRight",
-                }),
-              });
-            } else {
-              dispatch({
-                type: actions.SET_NOTIFY,
-                payload: null,
-              });
-            }
-          }
-        },
-        wallet: async (wallet: Wallet) => {
-          if (wallet.provider) {
-            const ethersProvider = new ethers.providers.Web3Provider(
-              wallet.provider
-            );
-            dispatch({ type: actions.SET_PROVIDER, payload: ethersProvider });
-            dispatch({
-              type: actions.SET_SIGNER,
-              payload: ethersProvider.getSigner(),
-            });
-            dispatch({
-              type: actions.SET_NETWORK,
-              payload: await ethersProvider.getNetwork(),
-            });
-          } else {
-            dispatch({ type: actions.SET_PROVIDER, payload: null });
-            dispatch({ type: actions.SET_NETWORK, payload: null });
-          }
-        },
-      };
-
-      connect(dispatch, network, subscriptions, onboard);
```

Comment:
because this is in a use effect with so many dependencies, i didnt trust that this was only being initialized once